src/query/service/src/interpreters/access/privilege_access.rs

```diff
@@ -99,6 +99,8 @@ impl AccessChecker for PrivilegeAccess {
                     .await?;
             }
             Plan::RenameTable(_) => {}
+            Plan::AddTableColumn(_) => {}
```

Comment:
Here should add privileges for Add/Drop table column and a SQL logic test for them.